src/ai/ai.cpp

```diff
@@ -1833,11 +1887,23 @@ bool will_accept_crisis_peace_offer(sys::state& state, dcon::nation_id to, dcon:
 
 void update_war_intervention(sys::state& state) {
 	for(auto& gp : state.great_nations) {
-		if(state.world.nation_get_is_player_controlled(gp.nation) == false && state.world.nation_get_is_at_war(gp.nation) == false) {
```

Comment:
Are you sure removing this is a good idea? I don't think that the AI should generally be jumping into more wars if it is already at war